source/docs/getting-started/installing.html.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "Installing"
+---
+
+##### Installing
+
+As this is a quick start guide, it doesn't assume any great familiarity with Chef or Ruby and takes you through the process of writing a Chef cookbook with automated testing as standard. In order to follow this guide, we'll need a few tools first.
+
+###### Pre-requisites
+
+- 64 bit operating system
+- CPU Virtualization enabled
+
+In order to virtualize a 64 bit operating system, one must also be running a 64 bit operating system. Most importantly, the CPU itself must support hardware virtualization extensions and this must be enabled in the BIOS/EFI. Most modern processors support virtualization extensions in the form of VT-x (Intel) or AMD-V (AMD).
+
+<div class="callout">
+<h3 class="callout--title">Windows and Hyper-V</h3>
```

Comment:
Given recent developments on the Packer and Hyper-V interest front (customers and community) I wanted to note this and eventually let the guide let this be interchangeable or set different guides per driver.